index.html

```diff
@@ -48,6 +48,156 @@ <h1 class="logo-text">Pascal</h1>
         </div>
     </section>
 
+    <section class="work">
+
```

Comment:
- [ ] Please try to stick to Figma design as much as possible. please change the background color of the section to `#f7f7f9`, currently it is too dark.
- [ ] Please give a little more margin around the left and right side of the cards
- [ ] Please delete box shadow from cards to meet the Figma design:


Your design  | Figma design
------------- | -------------
<img width="389" alt="Screenshot 2022-03-18 at 14 23 53" src="https://user-images.githubusercontent.com/77038610/158986033-f0a99951-784d-4a35-91f8-44a1ec3f10fd.png"> | <img width="345" alt="Screenshot 2022-03-18 at 14 23 31" src="https://user-images.githubusercontent.com/77038610/158985957-2528e002-7544-495e-83d0-f47ec3558480.png">


- [ ] Please add more space between cards

Your design  | Figma design
------------- | -------------
<img width="351" alt="Screenshot 2022-03-18 at 14 41 01" src="https://user-images.githubusercontent.com/77038610/158988637-40ffea76-c6c5-468c-947c-1c9692551a4d.png"> |  <img width="426" alt="Screenshot 2022-03-18 at 14 41 33" src="https://user-images.githubusercontent.com/77038610/158988685-3e0b79c3-d029-46e5-a4d8-333268d7a8a5.png">